src/Discord.Net.Rest/API/Common/InteractionResponse.cs

```diff
@@ -0,0 +1,13 @@
+using Newtonsoft.Json;
+
+namespace Discord.API
+{
+    [JsonObject(MemberSerialization = MemberSerialization.OptIn)]
+    internal class InteractionResponse
```

Comment:
duplicate of https://github.com/Discord-Net-Labs/Discord.Net-Labs/blob/Interactions/src/Discord.Net.Rest/API/Common/InteractionResponse.cs